src/index.js

```diff
@@ -7,17 +7,15 @@ import 'core-js/fn/set';
 import 'core-js/fn/number/is-integer';
 import 'core-js/es6/symbol';
 import 'core-js/fn/string/repeat';
-import {init} from './lib/init';
+import { init as initializeStore } from './lib/init';
 import log from "./lib/log";
 import config from "./lib/config";
 import { decodeConsentData, readConsentCookie } from "./lib/cookie/cookie";
 import {fetchGlobalVendorList} from "./lib/vendor";
 
 const TCF_CONFIG = '__tcfConfig';
 
-function handleConsentResult(tcfApi, store = {}, vendorList, consentData = {}) {
-	const { isConsentToolShowing } = store;
-
+const handleConsentResult = (tcfApi, vendorList, consentData = {}) => {
```

Comment:
You don't use `tcfApi` value anywhere inside this function

Reply:
removed